source/types.ts

```diff
@@ -328,7 +328,15 @@ export type AugmentedRequest = Request & {
  */
 export type RateLimitInfo = {
 	limit: number
-	current: number
+	used: number
 	remaining: number
 	resetTime: Date | undefined
+
+	/**
+	 * NOTE: The `current` field is deprecated and renamed to `used`. The library
+	 * will still set the `current` property, and you can still access it, but it
+	 * will be hidden from iteration and JSON.stringify calls. See:
+	 * https://github.com/express-rate-limit/express-rate-limit/discussions/372#discussioncomment-6915685
+	 */
+	// current: number
```

Comment:
Perhaps we should document it as optional and @deprecated?

```suggestion
   * @deprecated
	 */
	current?: number
```

I don't know, maybe your way is better. I don't think typescript has a good way of handling this.

Reply:
I didn't do the deprecation thing because we were hiding the fact that the property even existed, and we didn't want anyone to use it.